src/plugins/interactive_setup/public/use_form.ts

```diff
@@ -128,6 +128,7 @@ export function useFormState<Values extends FormValues, Result>({
   const [touched, setTouched] = useState<TouchedFields<Values>>({});
   const [submitCount, setSubmitCount] = useState(0);
 
+  // @ts-expect-error
```

Comment:
Nit: I personally prefer to add some text to this comment explaining what we're trying to ignore. At a minimum, I just copy the TS error verbatim if it's not too long